spec/controllers/posts_controllers_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
```

Comment:
* Kindly fix this error i get when i try to run your test files `TypeError: superclass must be an instance of Class (given an instance of Module)` below ⬇️ is a screenshot. HINT 💭 I think the error comes from your `postController class` instead `class PostsController < ApplicationController` you wrote `class PostsController < ActionController`
![image](https://user-images.githubusercontent.com/52497246/178389031-57eced86-fead-480e-953d-8e65e1c5589a.png)